server/src/main/java/org/elasticsearch/action/admin/indices/datastream/CreateDataStreamAction.java

```diff
@@ -64,7 +76,14 @@ public void setTimestampFieldName(String timestampFieldName) {
 
         @Override
         public ActionRequestValidationException validate() {
-            return null;
+            ActionRequestValidationException validationException = null;
```

Comment:
maybe also add a unit test for this method?
(this can be added to `CreateDataStreamRequestTests` class)